### DISCUSSION TITLE
Notifying adoption coordinators about medical treatments?

### DISCUSSION THREAD

**Discussion by jezwinters** [sheltermanager/asm3#1361] [Q&A / Help] [answered]
Hello!

My rescue has just started using shelter manager and I'm in the process of getting everything set up. I'm trying to figure out a way that we can notify our adoption coordinators of any medical treatments like deflea/worming that the animals they oversee need to have.

We'd rather not be sending emails to our foster carers directly as we prefer to have our adoption coordinators communicate with them personally. I'm struggling to figure out the best way to do this as medical treatments don't seem to get added to the diary so notifications don't go out that way, and I'd rather our adoption coordinators not have to check shelter manager every day to see what's due.

Can anyone give me advice on how I could set up a weekly email to each coordinator with any medical treatments for their animals?

Thanks so much!

**Comment by bobintetley** [comment]
It sounds like what you want is almost exactly what the "weekly fosterer email" function already does? It does email the foster carers directly though, since the animals are in their care and usually it will be the fosterer arranging to bring them into the shelter or take them to the vets to receive medical treatment. 

I'm sure your org works differently, but in most that we deal with the adoption coordinators tend to deal with applications and viewings rather than animal care directly?

**Reply by zam-can-code** [reply]
Where can the weekly fosterer email be found? Is that a report?

**Reply by jezwinters** [reply]
Yep! It's *almost* exactly what I'm after, so close but just not quite it. The way that we operate is that who I have in Shelter Manager as Adoption Coordinators are effectively our case managers, who deal with our foster carers and assist them with anything - like making sure they have deflea/worming medication.

Would I be able to make a mail merge that would send this kind of information? Even if I had to trigger it manually?

I'll admit I'm not the most proficient with SQL but I'm more than willing to learn if this is the way to do it.


@sedentarysam - the weekly foster email option is found under the medical tab of options

**Reply by bobintetley** [reply, marked as answer]
It would be really difficult to make a mail merge to include that information as there is a fair amount of data pulled into the weekly fosterer email and it uses a substantial amount of logic.

I think the simplest way that we might be able to accommodate this in future is with an option to send a copy of the weekly fosterer emails to the adoption coordinator as well (but not instead of).
